src/core/dev/cq_mgr_tx.cpp

```diff
@@ -214,6 +215,16 @@ int cq_mgr_tx::poll_and_process_element_tx(uint64_t *p_cq_poll_sn)
         if (unlikely(cqe->op_own & 0x80) && is_error_opcode(cqe->op_own >> 4)) {
             // m_p_cq_stat->n_tx_cqe_error++; Future counter
             log_cqe_error(cqe);
+
```

Comment:
This is not the right place to do it. This is only TCP logic and should not reside in cq_mgr which is a generic sq poller.

Reply:
We need to think how to propagate calls from cq_mgr to the related sockinfo. Today we dont have such mechanism. but seems like we need one. it will also be usefull for faireness handling. We can have a sockinfo pointer with some virtual method inside sq_wqe_prop but lets think if it wouldn't harm perf.